src/layers/legacy/medCoreLegacy/database/medLocalDbController.cpp

```diff
@@ -107,16 +107,30 @@ bool medLocalDbController::createStudyTable()
 {
     QSqlQuery query(m_database);
 
-    return query.prepare(
-               "CREATE TABLE IF NOT EXISTS study ("
-               " id        INTEGER      PRIMARY KEY,"
-               " patient   INTEGER," // FOREIGN KEY
-               " name         TEXT,"
-               " uid          TEXT,"
-               " thumbnail    TEXT,"
-               " studyId      TEXT"
-               ");") &&
+    query.prepare(
+        "CREATE TABLE IF NOT EXISTS study ("
+        " id        INTEGER      PRIMARY KEY,"
+        " patient   INTEGER," // FOREIGN KEY
+        " name         TEXT,"
+        " uid          TEXT,"
+        " thumbnail    TEXT,"
+        " studyId      TEXT,"
+        " time         TEXT,"
+        " date         TEXT"
+        ");") &&
            execQuery(query, __FILE__, __LINE__);
+
+    // Get all the information about the table columns
+    query.prepare("PRAGMA table_info(study)");
+    if (!execQuery(query, __FILE__, __LINE__))
+    {
+        qDebug() << DTK_COLOR_FG_RED << query.lastError() << DTK_NO_COLOR;
+    }
+
+    this->addTextColumnToStudyTableIfNeeded(query, "time");
+    this->addTextColumnToStudyTableIfNeeded(query, "date");
+
+    return true;
```

Comment:
the function never return an error whereas it was possible before. Is it the behavior you were searching to create?

Reply:
Same as previous comment

Reply:
same as previous comment